cdap-ui/app/features/tracker/directives/lineage/lineage.less

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@grey: #999;
+@font: #3d3d3d;
+@node-height: 55px;
+@node-width: 175px;
+
+my-lineage-diagram {
+  .lineage-diagram-container {
+    position: relative;
+    height: 100vh; // need to fix!!
```

Comment:
100vh doesn't take into account height of footer and header. Will need to come up with a way to dynamically get the height of the diagram